docs/platforms/kotlin-multiplatform/swift-package-manager.mdx

```diff
@@ -1,115 +1,16 @@
 ---
 title: Swift Package Manager (SPM)
-description: "Learn about installing the Sentry Kotlin Multiplatform SDK with Swift Package Manager."
+description: "Learn about installing the Sentry Kotlin Multiplatform SDK while using Swift Package Manager."
 sidebar_order: 2
 ---
 
 Swift Package Manager is a powerful tool for managing dependencies in Swift that allows developers to enjoy a more native integration experience with Xcode.
-If you already use Swift Package Manager or prefer it over other package managers on Apple platforms, this guide will show you how to install the Kotlin Multiplatform SDK using Swift Package Manager.
-
-<Note>
-Adding third party dependencies through Swift Package Manager in libraries such as the Sentry SDK is not yet officially supported in the Kotlin Multiplatform ecosystem and might not be compatible in more advanced use cases. We recommend [installing our SDK with Cocoapods](/platforms/kotlin-multiplatform/#install) for the best compatibility and experience with Kotlin Multiplatform.
-</Note>
-
-## Prerequisites
-
-Ensure compatibility in Kotlin Multiplatform projects targeting Cocoa by using the version of the Cocoa SDK specified in our [Cocoa SDK Version Compatibility Table](/platforms/kotlin-multiplatform/#cocoa-sdk-version-compatibility-table).
-
-### Export framework statically
-
-Make sure your Kotlin Multiplatform framework is exported as a static framework by adding `isStatic = true`
+If you already use Swift Package Manager or prefer it over other package managers on Apple platforms, this guide will show you how to install the Kotlin Multiplatform SDK while using Swift Package Manager.
```

Comment:
```suggestion
If you already use SPM or prefer it over other package managers on Apple platforms, this guide will show you how to install the Kotlin Multiplatform SDK while using Swift Package Manager.
```

Reply:
I feel like this doc no longer covers how to install the SDK... Not sure what the point of this doc is now. Maybe we should consider adding some SPM info to the top level getting started doc if the steps are the same?

Reply:
I think that it's definitely a better experience for the users to be able to get all the instructions on the same page as opposed to having to go back and forth. So if it's important to have a separate section highlighting SPM, it makes sense to keep the instructions :) 

The other option is to just move the introduction and the install information to the SPM section of the getting started page.